release-matrix/release-matrix.json

```diff
@@ -1,10 +1,18 @@
 [
   {
-    "mkeReleaseVersion": "v4.0.1-alpha.1",
+    "mkeReleaseVersion": "v4.0.1-alpha.2",
     "blueprintOperatorVersion": "v1.0.18",
-    "mkeOperatorVersion": "v1.2.0",
+    "mkeOperatorVersion": "v1.2.1",
     "k0sVersion": "v1.31.1+k0s.1",
     "isLatest": true,
+    "minimumMkectlVersion": "v4.0.1-alpha.2"
+  },
+  {
+    "mkeReleaseVersion": "v4.0.1-alpha.1",
```

Comment:
should we still have this one on the matrix, even though its broken?

Reply:
I don't mind having it removed since we don't want anyone deploying it.

Reply:
removed